src/app/newSettings/Sections/Workspace/Logs/AccessLogsSection.tsx

```diff
@@ -0,0 +1,255 @@
+  const [searchMembersInputValue, setSearchMembersInputValue] = useState<string>('');
+  const [daysFilter, setDaysFilter] = useState<number | undefined>();
+  const [activityFilter, setActivityFilter] = useState<WorkspaceLogType[]>([]);
+  const [orderBy, setOrderBy] = useState<{ key: 'updatedAt' | 'type' | 'platform'; direction: 'ASC' | 'DESC' }>({
+    key: 'updatedAt',
+    direction: 'DESC',
+  });
+  const debouncedSearchMemberValue = useDebounce(searchMembersInputValue, 500);
+  const { accessLogs, isLoading, hasMoreItems, loadMoreItems } = useAccessLogs({
+    activity: activityFilter,
+    lastDays: daysFilter,
+    member: debouncedSearchMemberValue,
+    orderBy: [orderBy.key, orderBy.direction].join(':'),
+  });
+
+  function getActivityType(type: WorkspaceLogType) {
+    const getLogKey = getEnumKey(WorkspaceLogType, type);
+    return translate(`preferences.workspace.accessLogs.filterActions.activity.${getLogKey}`) || 'Unknown action';
+  }
+
+  function getPlatformType(platform: WorkspaceLogPlatform) {
+    const getPlatformKey = getEnumKey(WorkspaceLogPlatform, platform);
+    return translate(`preferences.workspace.accessLogs.filterActions.platform.${getPlatformKey}`) || 'Unknown';
+  }
+
+  const handleActivityFilters = (actionType: WorkspaceLogType) => {
+    const isFilterActivated = activityFilter?.some((activity) => activity === actionType);
+    const newActivityFilters = isFilterActivated
+      ? activityFilter.filter((activity) => activity !== actionType)
+      : [...activityFilter, actionType];
+
+    setActivityFilter(newActivityFilters);
+  };
+
+  const handleDaysFilter = (days: number) => {
+    setDaysFilter(days);
+  };
+
+  const onClearAllFilters = () => {
+    setSearchMembersInputValue('');
+    setDaysFilter(undefined);
+    setActivityFilter([]);
+    setOrderBy({
+      key: 'updatedAt',
+      direction: 'DESC',
+    });
+  };
+
+  const formatDate = (updatedAt: Date) => {
+    const formatted = dateService.format(updatedAt, 'MMM D, YYYY');
+    return formatted.charAt(0).toUpperCase() + formatted.slice(1);
+  };
+
+  const formatTime = (updatedAt: Date) => dateService.format(updatedAt, 'hh:mm A');
+
+  const headerList: HeaderItemsProps[] = [
+    {
+      title: translate('preferences.workspace.accessLogs.headerTable.date'),
+      isSortByAvailable: true,
+      width: '165px',
+      sortKey: 'updatedAt',
+      defaultSort: 'ASC',
+    },
+    {
+      title: translate('preferences.workspace.accessLogs.headerTable.member'),
+      isSortByAvailable: false,
+      width: '191px',
+    },
+    {
+      title: translate('preferences.workspace.accessLogs.headerTable.activity'),
+      isSortByAvailable: true,
+      sortKey: 'type',
+      width: '161px',
+      defaultSort: 'ASC',
+    },
+    {
+      title: translate('preferences.workspace.accessLogs.headerTable.access'),
+      isSortByAvailable: true,
+      sortKey: 'platform',
+      width: '120px',
```

Comment:
Why not use same width in all columns and here is in px? Add a width for everything in some constant and it is much easier to maintain this component